.github/workflows/wasm.yml

```diff
@@ -16,14 +15,55 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - name: Setup rust toolchain
-        uses: dtolnay/rust-toolchain@master
+      - name: Setup Nix
+        uses: cachix/install-nix-action@v20
         with:
-          toolchain: 1.66.0
+          nix_path: nixpkgs=channel:nixos-22.11
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install wasm-pack
-        run: cargo install wasm-pack
-
-      - name: Build wasm crate
+      - name: Build wasm crate with Nix
         working-directory: ./crates/wasm
-        run: ./build-wasm
+        run: nix-build wasm.nix
+
+      - name: Checkout noir-wasm-testing
+        uses: actions/checkout@v3
+        with:
+          repository: noir-lang/noir-wasm-testing
+          path: noir-wasm-testing
+
+      - name: Install jq
+        run: sudo apt-get install -y jq
+
+      - name: Inject built wasm into noir-wasm-testing
+        run: |
+          cp -r ./crates/wasm/result ./noir-wasm-testing/noir_wasm
+          jq '.dependencies.noir_wasm = "file:./noir_wasm"' ./noir-wasm-testing/package.json > ./noir-wasm-testing/package.json.tmp
+          mv ./noir-wasm-testing/package.json.tmp ./noir-wasm-testing/package.json
+
+      - name: Run build.sh in noir-wasm-testing
+        working-directory: ./noir-wasm-testing
+        run: |
+          chmod +x build.sh
+          ./build.sh
```

Comment:
Is the `nix-build wasm.nix` command not running this script?

Reply:
I don't think it is. This is the `build.sh` of `noir-wasm-testing`, not the one of `noir_wasm`